### ISSUE TITLE
Citing pdbfixer

### ISSUE THREAD

**Issue by jchodera** [pandegroup/pdbfixer#137]
Do you guys have any preference on how we should cite PDBFixer?

Have you considered enabling [Zenodo integration](https://zenodo.org/account/settings/github/) to give releases citable DOIs?

**Comment by peastman** [comment]
Never heard of it before.

**Comment by jchodera** [comment]
Let me rephrase:

Would you consider enabling [Zenodo integration](https://zenodo.org/account/settings/github/) to give releases citable DOIs?
